packages/eas-build-job/src/metadata.ts

```diff
@@ -198,7 +193,6 @@ export const MetadataSchema = Joi.object({
   message: Joi.string().max(1024),
   runFromCI: Joi.boolean(),
   runWithNoWaitFlag: Joi.boolean(),
-  buildMode: Joi.string().valid('build', 'resign', 'custom'),
```

Comment:
It will break builds is turtle-api https://github.com/expo/turtle-v2/blob/6fd575033d373ab443138a7552e04a767a0672c9/src/services/turtle-api/src/api/builds.ts#L500

Reply:
For future readers — it won't because `stripUnknown: true` https://github.com/expo/eas-build/blob/23e76f86b6152596ad28eb798a64ff3f50fdc212/packages/eas-build-job/src/metadata.ts#L205-L209